webapp/channels/src/components/post/post_component.test.tsx

```diff
@@ -324,4 +324,139 @@ describe('PostComponent', () => {
             });
         });
     });
+
+    describe('file list', () => {
+        test('should show file list in post', () => {
+            const fileInfo1 = TestHelper.getFileInfoMock({id: 'fileId1', name: 'file1.jpg'});
+            const fileInfo2 = TestHelper.getFileInfoMock({id: 'fileId2', name: 'file2.jpg'});
+            const fileInfo3 = TestHelper.getFileInfoMock({id: 'fileId3', name: 'file3.jpg'});
+
+            const post = TestHelper.getPostMock({file_ids: [fileInfo1.id, fileInfo2.id, fileInfo3.id]});
+
+            const state: DeepPartial<GlobalState> = {
+                entities: {
+                    posts: {
+                        posts: {
+                            [post.id]: post,
+                        },
+                    },
+                    files: {
+                        files: {
+                            [fileInfo1.id]: fileInfo1,
+                            [fileInfo2.id]: fileInfo2,
+                            [fileInfo3.id]: fileInfo3,
+                        },
+                        fileIdsByPostId: {
+                            [baseProps.post.id]: ['fileId1', 'fileId2', 'fileId3'],
+                        },
+                    },
+                },
+            };
+
+            const props = {
+                ...baseProps,
+                post,
+            };
+
+            const {container} = renderWithContext(<PostComponent {...props}/>, state);
+            expect(screen.getByTestId('fileAttachmentList')).toBeInTheDocument();
+            expect(container.querySelectorAll('.post-image__column')).toHaveLength(3);
```

Comment:
is `querySelectorAll` not available in screen?

Reply:
No, screen only has the selectors for text content, test id etc. The more powerful selectors are in `container`.